test/e2e/lib/helpers/broker_event_transformation_test_with_source.go

```diff
@@ -63,7 +63,24 @@ func BrokerEventTransformationTestHelper(client *lib.Client, brokerURL url.URL,
 	// Create a target Job to receive the events.
 	makeTargetJobOrDie(client, targetName)
 
-	createTriggersAndKService(client, brokerName, targetName)
+	// Create the Knative Service.
+	kserviceName := CreateKService(client, "receiver")
+
+	// Create a Trigger with the Knative Service subscriber.
+	triggerFilter:=eventingtestresources.WithAttributesTriggerFilter(
+		eventingv1alpha1.TriggerAnyFilter, eventingv1alpha1.TriggerAnyFilter,
+		map[string]interface{}{"type": lib.E2EDummyEventType})
+	createTriggerWithKServiceSubscriber(client, brokerName, kserviceName, triggerFilter)
+
+	// Create a Trigger with the target Service subscriber.
+	respTriggerFilter:=eventingtestresources.WithAttributesTriggerFilter(
+		eventingv1alpha1.TriggerAnyFilter, eventingv1alpha1.TriggerAnyFilter,
+		map[string]interface{}{"type": 	lib.E2EDummyRespEventType})
+	createTriggerWithTargetServiceSubscriber(client, brokerName, targetName, respTriggerFilter)
+
+	// Wait for ksvc, trigger ready.
+	client.Core.WaitForResourceReadyOrFail(kserviceName, lib.KsvcTypeMeta)
```

Comment:
Yep. There is a WaitForResource**s**ReadyOrFail. 
But WaitForResource**s**ReadyOrFail also needs to specify `typemeta *metav1.TypeMeta`. Here we use WaitForResourceReadyOrFail is because we only have one ksvc.
For the triggers, we use WaitForResource**s**ReadyOrFail  because we have two triggers